### DISCUSSION TITLE
HotChocolate.Stitching.Redis v14

### DISCUSSION THREAD

**Discussion by arrumanov** [ChilliCream/hotchocolate#7641] [General]
When is the v14 release planned for HotChocolate.Stitching.Redis?

**Comment by arrumanov** [comment]
Stitching was deprecated in 13 and now is fully replaced by fusion
https://hotchocolategraphql.slack.com/archives/C04SQUPC82W

**Comment by jkears** [comment]
This is a breaking change with no resolution |An error occurred while accessing the Microsoft.Extensions.Hosting services. Continuing without the application service provider. Error: Derived method 'OnAfterResolveRootType' in type 'HotChocolate.Stitching.SchemaDefinitions.SchemaDefinitionTypeInterceptor' from assembly 'HotChocolate.Stitching, Version=13.9.14.0, Culture=neutral, PublicKeyToken=null' cannot reduce access.| ... how can we ever upgrade to new releases when this type of changes happens pn a regular basis ... every time!!!

Please explain how I am to publish schema in my 14.x app now that it now longer exists?????

```
 public static IServiceCollection AddGraphQlServer(this IServiceCollection serviceCollection)
                    {
                        var requestExecutorBuilder = serviceCollection.AddGraphQLServer()
             				.AddAuthorization()
             				.AddType<ErrorResponse>()
             				.AddType<AggregateEvent>()              
             				
             				.AddProjections()
							.AddFiltering()
							.AddSorting()
             				
							.RegisterDbContextFactory<AppDbContextUOW>()
             				.PublishSchemaDefinition(c =>
                                   {
                                       c.SetName("modelservices");  
                                       c.PublishToRedis("nextwarecoreservicesmetagw", sp => sp.GetRequiredService<IConnectionMultiplexer>());
                                   })
             				.InitializeOnStartup()
             				// Uncomment to flow exceptions=> .ModifyRequestOptions(opt => opt.IncludeExceptionDetails = true) 
             				.AddInMemorySubscriptions();
                        
                        AddGraphQlTypes(serviceCollection, requestExecutorBuilder);
                        return serviceCollection;
                    }

```
